### ISSUE TITLE
Issue severity not shown in tty output

### ISSUE THREAD

**Issue by huornlmj** [hadolint/hadolint#495]
- [ ] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
Issue severity should be included in default tty output

### Actual behavior
No issue severity included in default tty output.  Severity is included however in all of the other output types (i.e. json, checkstyle, codeclimate, codacy).

### Steps to reproduce the behavior

Scan the hadolint example Dockerfile using command defaults (i.e. tty output)

```Dockerfile
FROM debian
RUN export node_version="0.10" \
&& apt-get update && apt-get -y install nodejs="$node_verion"
COPY package.json usr/src/app
RUN cd /usr/src/app \
&& npm install node-static
EXPOSE 80000
CMD ["npm", "start"]
```

```
$ hadolint Dockerfile.example
Dockerfile.example:1 DL3006 Always tag the version of an image explicitly
Dockerfile.example:2 SC2154 node_verion is referenced but not assigned (did you mean 'node_version'?).
Dockerfile.example:2 DL3009 Delete the apt-get lists after installing something
Dockerfile.example:2 DL3015 Avoid additional packages by specifying `--no-install-recommends`
Dockerfile.example:5 DL3003 Use WORKDIR to switch to a directory
Dockerfile.example:5 DL3016 Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`
Dockerfile.example:7 DL3011 Valid UNIX ports range from 0 to 65535
```

Note that no issue severity it included.  Now run the same and use a different output method (e.g. json)

```json
$ hadolint Dockerfile.example -f json
[
{
    "line": 1,
    "code": "DL3006",
    "message": "Always tag the version of an image explicitly",
    "column": 1,
    "file": "Dockerfile.example",
    "level": "warning"
},
{
    "line": 2,
    "code": "SC2154",
    "message": "node_verion is referenced but not assigned (did you mean 'node_version'?).",
    "column": 1,
    "file": "Dockerfile.example",
    "level": "warning"
},
{
    "line": 2,
    "code": "DL3009",
    "message": "Delete the apt-get lists after installing something",
    "column": 1,
    "file": "Dockerfile.example",
    "level": "info"
},
{
    "line": 2,
    "code": "DL3015",
    "message": "Avoid additional packages by specifying `--no-install-recommends`",
    "column": 1,
    "file": "Dockerfile.example",
    "level": "info"
},
{
    "line": 5,
    "code": "DL3003",
    "message": "Use WORKDIR to switch to a directory",
    "column": 1,
    "file": "Dockerfile.example",
    "level": "warning"
},
{
    "line": 5,
    "code": "DL3016",
    "message": "Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`",
    "column": 1,
    "file": "Dockerfile.example",
    "level": "warning"
},
{
    "line": 7,
    "code": "DL3011",
    "message": "Valid UNIX ports range from 0 to 65535",
    "column": 1,
    "file": "Dockerfile.example",
    "level": "error"
}]
```

Note that the severity is shown as 'level'.

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**
```
$ hadolint -v
Haskell Dockerfile Linter v1.18.2-0-gf75d690
```
